Dockerfile

```
@@ -11,12 +15,19 @@ RUN yum install -y \
 	db4-utils \
 	db4 && yum clean all
 
+RUN usermod -u ${USER_ID} ftp
+RUN groupmod -g ${GROUP_ID} ftp
+
 ENV FTP_USER **String**
 ENV FTP_PASS **Random**
 ENV PASV_ADDRESS **IPv4**
+ENV PASV_ADDR_RESOLVE No
+ENV PASV_ENABLE Yes
 ENV PASV_MIN_PORT 21100
 ENV PASV_MAX_PORT 21110
 ENV LOG_STDOUT **Boolean**
+ENV FILE_OPEN_MODE 0700
+ENV LOCAL_UMASK 0002
```

May I ask for more context on this?

---

What exactly is unclear here?

---

I guess we need to use default options provided by vsftpd docs.

> file_open_mode
The permissions with which uploaded files are created. Umasks are applied on top of this value. You may wish to change to 0777 if you want uploaded files to be executable.
Default: 0666

> local_umask
The value that the umask for file creation is set to for local users. NOTE! If you want to specify octal values, remember the "0" prefix otherwise the value will be treated as a base 10 integer!
Default: 077

[ref](https://security.appspot.com/vsftpd/vsftpd_conf.html)

```suggestion
ENV FILE_OPEN_MODE 0666
ENV LOCAL_UMASK 077
```

---

Not sure what I have to do?! 

merge what into where?

---

@mindrunner 
Try to merge this pr https://github.com/mindrunner/docker-vsftpd/pull/1
And then https://github.com/fauria/docker-vsftpd/pull/16 will be updated automatically.